### PR TITLE
prepare rel. 1.1

### PR DESCRIPTION
**Pull request by exaexa**
there's plenty of minor stuff that can still be cleaned up, this PR is grouping that stuff.

### CODE REVIEW

File: R/histogram_to_rgbwt.R [hunk 1]
```diff
@@ -20,42 +20,45 @@
...
+    pal_size * (fhistogram - zlim[1]) / max((zlim[2] - zlim[1]), scattermore.globals$epsilon)
+  ))
 
   result <- .C("histogram_to_rgbwt",
-    dimen = as.integer(c(rows, cols, size)),
+    dimen = as.integer(c(rows, cols, pal_size)),
     fRGBWT = as.single(RGBWT),
     RGBA = as.single(RGBA / 255),
-    normalized_fhistogram = as.single(normalized_fhistogram)
+    normalized_fhistogram = as.integer(normalized_fhistogram)
```
Comment (teri934) [inline R/histogram_to_rgbwt.R:60]: should't there be as.single? i am not sure if we should change the type parameter for histogram... the result in one example isn't correct, it should be a colorful image... so i think there could be a mistake?
image

File: src/histogram_to_rgbwt.cpp
Comment (teri934) [inline src/histogram_to_rgbwt.cpp:1]: not sure about deleting the bin to determine the column in the palette... but i think something in .R or .cpp file histogram_to_rgbwt is wrong

**Review by teri934 — CHANGES_REQUESTED** [review]
the histogram functionality isn't correct, the result is different

File: R/histogram_to_rgbwt.R (hunk 1 above)
Comment (exaexa) [reply R/histogram_to_rgbwt.R:60]: I actually vastly simplified the default colors so yeah, that's correct.

By converting to integers directly in R I avoid the size_t UB later.

Comment (exaexa) [reply R/histogram_to_rgbwt.R:61]: anyway the colors are back (just wanted to avoid specifying custom ones so I'm going with the hcl.colors)

Comment (exaexa) [reply R/histogram_to_rgbwt.R:61]: (current version) 
image

Comment (teri934) [reply R/histogram_to_rgbwt.R:61]: ou, okay, this seems correct, I got confused that the default parameters were different :D

**Review by teri934 — APPROVED** [review]

**Pull request merged**